src/app/app.component.ts

```diff
@@ -312,6 +314,14 @@ export class AppComponent implements OnInit {
                         break;
                     case 'sharingPaywall':
                         this.isSharingPaywallDisplay = true;
+                        break;
+                    case 'navigate':
+                        if (message.goBack) {
+                            this.location.back();
```

Comment:
💡  Note that this may not work on mobile if we use the HttpServer as the navigation history is broken in that case. But as we deactivated HttpServer for cozy-pass this is not problematic